submissions/iva-stasia/html-css-popup/index.html

```diff
@@ -0,0 +1,68 @@
+              <ul class="popup">
+                <li class="p_item"><a href="#" class="p_icon calendar"><span class="caption">Calendar</span></a></li>
+                <li class="p_item"><a href="#" class="p_icon mail"><span class="caption">Mail</span></a></li>
+                <li class="p_item"><a href="#" class="p_icon news"><span class="caption">News</span></a></li>
+                <li class="p_item"><a href="#" class="p_icon map"><span class="caption">Maps</span></a></li>
+                <li class="p_item"><a href="#" class="p_icon file"><span class="caption">Files</span></a></li>
+                <li class="p_item"><a href="#" class="p_icon contact"><span class="caption">Contacts</span></a></li>
+                <li class="p_item"><a href="#" class="p_icon translate"><span class="caption">Translate</span></a></li>
+                <li class="p_item"><a href="#" class="p_icon video"><span class="caption">Videos</span></a></li>
+                <li class="p_item"><a href="#" class="p_icon search"><span class="caption">Search</span></a></li>
+                <li class="p_item"><a href="#" class="p_icon photo"><span class="caption">Photos</span></a></li>
+                <li class="p_item"><a href="#" class="p_icon apps"><span class="caption">Apps</span></a></li>
+                <li class="p_item"><a href="#" class="p_icon doc"><span class="caption">Docs</span></a></li>
+                <li class="details">
+                   <details>
+                    <summary class="more_button">More</summary><hr class="line">
+                    <ul class="more">
+                      <li class="m_item"><a href="#" class="m_icon chat"><span class="caption">Chat</span></a></li>
+                      <li class="m_item"><a href="#" class="m_icon book"><span class="caption">Books</span></a></li>
+                      <li class="m_item"><a href="#" class="m_icon music"><span class="caption">Music</span></a></li>
+                      <li class="m_item"><a href="#" class="m_icon weather"><span class="caption">Weather</span></a></li>
+                      <li class="m_item"><a href="#" class="m_icon support"><span class="caption">Support</span></a></li>
+                      <li class="m_item"> 
+                        <label>
+                        <input type="checkbox" class="no_input" />
+                        <span class="m_icon dont_press"><span class="caption">Don't Click</span></span>
+                        <img class="no_img" src="./Images/edvard-munch-1.png" alt="" />
+                        </label>
+                      </li>
+                    </ul>  
+                  </details>
+                </li>
+              </ul>
+            </li>
+            <li class="nav_item"><a href="#3" class="nav_icon bell"></a></li>
+            <li class="nav_item"><a href="#4" class="nav_icon profile"></a></li>
+          </ul>
+        </nav>
+      </header>
+      <main class="main">
+      </main>
```

Comment:
Avoid unused tags in your code.

Reply:
I used this tag to style and position the header and footer using flex. However, this is not necessary now.
Changed the way these elements were positioned without using flex and removed the empty main tag.